### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by janisozaur**
For @spacek531

### CODE REVIEW

**Comment by janisozaur** [comment]
I have not yet seen it pass, the initial action to build and populate vcpkg cache takes an hour.

File: src/renderer.c [hunk 1]
```diff
@@ -409,7 +411,7 @@ framebuffer.fragments=malloc(framebuffer.width*framebuffer.height*sizeof(fragmen
 
 
 //Transform lights for view
-light_t transformed_lights[context->num_lights];
+light_t *transformed_lights = malloc(context->num_lights * sizeof(light_t));
```
Comment (janisozaur) [inline src/renderer.c:414]: Not freed

**Comment by janisozaur** [comment]
https://github.com/janisozaur/libIsoRender/actions/runs/127223185

the action is passing now, just need to free the buffer that's leaking.

File: CMakeLists.txt [hunk 2]
```diff
@@ -2,14 +2,27 @@ cmake_minimum_required (VERSION 3.1)
 project (IsoRender C)
 
 find_package(PNG REQUIRED)
```
Comment (spacek531) [inline CMakeLists.txt:4]: my vcpkg puts library is under a different name

```suggestion
find_package(libpng REQUIRED)
```

**Pull request merged**